assets/js/services/testing-helpers.js

```diff
@@ -50,7 +51,7 @@ export function renderWithRouterApollo(
   } = {}
 ) {
   const Wrapper = ({ children }) => (
-    <MockedProvider mocks={mocks} addTypename={false}>
+    <MockedProvider mocks={mocks} addTypename={false} resolvers={resolvers}>
```

Comment:
@kdid So this makes it work with tests eh?   Nice.

Reply:
It was throwing an error until I googled and added that. Not sure i totally understand how that's all working.